src/storage/sql/migrations/2020-07-25-065258_create_snippets/up.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE snippets (
+    -- an internal autoincrementing identifier used in foreign keys.
+    -- Normally not visible publicly, except for the case when it is
+    -- used for looking legacy snippets up by id
+    id SERIAL PRIMARY KEY,
+    -- a short unique snippet identifier visible to users
+    slug TEXT NOT NULL,
```

Comment:
**Q:** Should we limit `slug` by, let's say, 100 characters or use some `VARCHAR` instead? Or you think `TEXT` is simpler and preferable in our case?

Reply:
AFAIK, it makes no difference for efficiency, but we could use VARCHAR(N) as a sanity check. I think, 100 is way too high, though (as long we use both lowecase and uppercase letters + 0-9 digits in our range of allowed symbols).